build/templates/dotnet-install.yml

```diff
@@ -0,0 +1,9 @@
+steps:
+  - task: UseDotNet@2
```

Comment:
Do we really need to install a dependencies? Aren't they already loaded on the hosted agent?
@vincentcastagna

Reply:
Yeah the mobile workloads aren't install by default. I also made it like a MS engineer said to do for now on a developer blog.

https://devblogs.microsoft.com/dotnet/devops-for-dotnet-maui/